recipes/dev/manifest.json

```diff
@@ -1,10 +1,17 @@
 {
     "bundles": {
+        "Karser\\Recaptcha3Bundle\\KarserRecaptcha3Bundle": [
+            "all"
+        ],
```

Comment:
This bundle has no recipe?
Maybe we can add it directly on the recipes repos?

Reply:
It seems to use auto-generated recipe : https://github.com/karser/KarserRecaptcha3Bundle#installation

Reply:
It's not the auto-generated recipe.
https://github.com/symfony/recipes-contrib/tree/master/karser/karser-recaptcha3-bundle/0.1